### DISCUSSION TITLE
Admin Panel Fails on Safari

### DISCUSSION THREAD

**Discussion by beschneider** [directus/directus#10794] [Archive]
I just installed a fresh copy of Directus version 9.4.2 on my local machine (Mac M1). When I go to open the admin panel in Safari (localhost:8055/admin), I get a blank page, and the JavaScript console shows an SSL error (see picture). In Chrome, the admin panel works fine.

I have a different project on my local machine that is a little older version of Directus, 9.2.1. This project works just fine in Safari, and the .env files between the projects are identical (except database username/pw).

Any idea why this is happening?

<img width="1062" alt="Screen Shot 2021-12-31 at 1 29 54 PM" src="https://user-images.githubusercontent.com/75515236/147837138-71d775e1-fb07-44fb-b5ae-c062aedbb094.png">

**Comment by gearoidc** [comment]
Im getting the same issue when I upgraded to 9.4 from 9.0.0. http is redirecting to https. I get the same error across all browsers.

```
Failed to load resource: net::ERR_SSL_PROTOCOL_ERROR
```

**Comment by azrikahar** [comment]
@beschneider @gearoidc would you both mind trying this solution: https://github.com/directus/directus/issues/10806#issuecomment-1003876201? It should definitely be something specific to `9.4.2` as you mentioned.

**Reply by gearoidc** [reply]
> @beschneider @gearoidc would you both mind trying this solution: [#10806 (comment)](https://github.com/directus/directus/issues/10806#issuecomment-1003876201)? It should definitely be something specific to `9.4.2` as you mentioned.

That worked a treat @azrikahar thanks very much.

**Reply by azrikahar** [reply]
Thank you for promptly confirming this as well! Cheers
